content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Memperluas Kubernetes Cluster Anda
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kubernetes sangat mudah dikonfigurasi dan dikembangkan. Sehingga,
```

Comment:
```suggestion
Kubernetes sangat mudah dikonfigurasi dan diperluas. Sehingga,
```

Please be consistent with the translation throughout the page. Choose between one of "memperluas" or "mengembangkan".

Reply:
done